src/Renraku-Tests/ReMethodSourceCleanerTest.class.st

```diff
@@ -7,8 +7,117 @@ Class {
 	#category : #'Renraku-Tests-Utilities'
 }
 
-{ #category : #tests }
+{ #category : #'private - test data' }
+ReMethodSourceCleanerTest >> aleadyCleanedMethodStringKeepingDot [
+	^'dirtyMethodWithUnnecessarySpacesAndFinalDot
+	"Do not format this method - it by intention should include unnecessary spaces and a final dot"
+
+	|temp|
+
+	temp := 3 + 4.'
+]
+
+{ #category : #'private - test data' }
+ReMethodSourceCleanerTest >> alreadyCleanedMethodString [
+	^'dirtyMethodWithUnnecessarySpacesAndFinalDot
+	"Do not format this method - it by intention should include unnecessary spaces and a final dot"
+
+	|temp|
+
+	temp := 3 + 4'
+]
+
+{ #category : #'private - test data' }
+ReMethodSourceCleanerTest >> dirtyMethodWithUnnecessarySpacesAndFinalDot [
+	"Do not format this method - it by intention should include unnecessary spaces and a final dot"
+
+	|temp|
+
+	temp := 3 + 4
+]
+
+{ #category : #'private - test data' }
+ReMethodSourceCleanerTest >> properMethodWithFinalDot [
+	"This is a method without any unnecessary separators in the method body but a final dot"
+
+	^ 3 + 4.
```

Comment:
Maybe add a warning in the comment to prevent people to format this method

Reply:
Made it more clear in https://github.com/astares/pharo/commit/4904ac8610c2f07e369be13a25efb174b07400ba